test/interval_tests/numeric.jl

```diff
@@ -344,15 +344,8 @@ end
 @testset "Mince for `Interval`s" begin
     v = mince(-1 .. 1, 4)
     @test v == [-1 .. -0.5, -0.5 .. 0, 0 .. 0.5, 0.5 .. 1]
-    h = hull(hull(v[1], v[2]), hull(v[3], v[4]))
-    @test h == -1 .. 1
+    @test hull(v...) == -1 .. 1
```

Comment:
We can define `hull` for a `Vector` of intervals as well.

Reply:
Done in the last commit.